src/Microsoft.Framework.Localization/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Framework.Localization.Test")]
```

Comment:
In this case it's just an internal method to clear a global static cache, not types.